packages/@glimmer/syntax/test/loc-node-test.ts

```diff
@@ -349,10 +353,16 @@ foo"
 
     if (assertNodeType(dataDerpValue, 'ConcatStatement')) {
       let [fooStaticText, concat, huzzahStaticText] = dataDerpValue.parts;
-      locEqual(fooStaticText, 9, 14, 10, 0);
+      locEqual(fooStaticText, 9, 13, 10, 0);
```

Comment:
The prior assertion was wrong :sob:, and this test should have _already_ been failing